src/rules/selector-no-combinator/README.md

```diff
@@ -0,0 +1,35 @@
+# selector-no-combinator
+
+Disallow combinators.
```

Comment:
This should be "Disallow combinators in selectors" to match the description conventions.